src/splight_agent/engine.py

```diff
@@ -114,7 +109,12 @@ def _load_image(
         return image
 
     def _run_container(
-        self, image: Image, environment: dict, runspec: dict, labels: dict
+        self,
+        image: Image,
+        environment: dict,
+        runspec: dict,
+        labels: dict,
+        restart_policy: dict,
```

Comment:
restart_policy should be a value of the enum, not a dict.

Reply:
Docker sdk for python takes a dict as restart policy:
`
restart_policy (dict) –

Restart the container when it exits. Configured as a dictionary with keys:

Name One of on-failure, or always.

MaximumRetryCount Number of times to restart the container on failure.
`